packages/relay/src/db/schema.ts

```diff
@@ -142,6 +142,38 @@ const _schema = [
             },
         ],
     },
+    {
+        name: 'ReportHistory',
+        primaryKey: 'reportId',
+        rows: [
+            ['reportId', 'String'],
+            ['type', 'Int'], // 0: Post, 1: Comment
+            ['objectId', 'String'], // PostId or CommentId
+            ['reportorEpochKey', 'String'], // Epoch Key of the person who reported
+            ['reportorClaimedRep', 'Bool'], // TRUE: claimed, FALSE: not claimed
+            ['respondentEpochKey', 'String'], // Epoch Key of the person who was reported
+            ['respondentClaimedRep', 'Bool'], // TRUE: claimed, FALSE: not claimed
+            ['reason', 'String'], // Reason of the report
+            ['adjudicateCount', 'Int'], // Number of voters
+            {
```

Comment:
refer to the example from anondb https://github.com/vimwitch/anondb/blob/ed82c0a7a11a315344f19c0d79ddfb5f6edf5467/examples/schema.ts#L252-L269, setting the type as Object is enough, and this field should be set to optional.